deployments/charts/kuma/README.md

```diff
@@ -88,9 +88,10 @@ A Helm chart for the Kuma Control Plane
 | cni.image.registry | string | `"docker.io/kumahq"` | CNI image registry |
 | cni.image.repository | string | `"install-cni"` | CNI image repository |
 | cni.image.tag | string | `"0.0.10"` | CNI image tag |
-| cni.imageExperimental | object | `{"repository":"kuma-cni","tag":null}` | use new CNI image (experimental) |
-| cni.imageExperimental.repository | string | `"kuma-cni"` | CNI experimental image repository |
-| cni.imageExperimental.tag | string | `nil` | CNI experimental image tag - defaults to .Chart.AppVersion |
+| cni.delayStartupSeconds | int | `0` | it's only useful in tests to trigger a possible race condition |
```

Comment:
Kind of a nit but IMO this would be less ugly with a `cni.prestartCommand` that we then set to `sleep x`

Reply:
It's also used for `initialDelaySeconds` of the readiness probe so if it's just a nitpick I'm leaving this as it is. If you have an approach that could work for that as well - please comment.

Reply:
Maybe just allow general configuration of the readiness probe? Adding something "for testing only" to the public interface of something always feels like a code smell to me but I guess not bad enough to hold up the PR.